plugin.rb

```diff
@@ -85,12 +108,40 @@ class Engine < ::Rails::Engine
     scope.is_staff?
   end
 
+  add_to_class(:topic, :is_category_expert_question?) do
+    custom_fields[CategoryExperts::TOPIC_IS_CATEGORY_EXPERT_QUESTION] &&
+      !category.custom_fields[CategoryExperts::CATEGORY_EXPERT_GROUP_IDS].blank?
+  end
+
+  add_to_serializer(:topic_list_item, :needs_category_expert_post_approval) do
```

Comment:
You could save some space on our JSON by having this return `true` and only `include_`ing it if this condition is true.